scss/components/_button.scss

```diff
@@ -136,12 +136,12 @@ $button-opacity-disabled: 0.25 !default;
 }
 
 /// Adds disabled styles to a button by fading the element, reseting the cursor, and disabling pointer events.
-@mixin button-disabled {
+@mixin button-disabled($color: $primary-color) {
   opacity: $button-opacity-disabled;
   cursor: not-allowed;
 
   &:hover, &:focus {
-    background-color: $button-background;
+    background-color: $color;
     color: $button-color;
```

Comment:
Shouldn't this use `foreground($color, $button-color-alt, $button-color)`? if the button background color is very light? The secondary color is usually gray so we don't want the font color to be white then...